benchmarks/README.md

```diff
@@ -0,0 +1,99 @@
+# PermaplanT Benchmarks
+
+## Requirements
+
+-   nodejs 19.4.0
+-   npm
+
+## Installation and Usage
+
+1. Install dependencies
+
+```shell
+npm install
+```
+
+2. Populate the database with example maps
+
+In order to benchmark the performance of the canvas, we need to populate the database with example maps.
+There are maps of different sizes and complexity, which are stored in a single CSV file called `maps.csv` in the `/benchmarks` folder.
+The maps are stored in a CSV file because they are huge and it is easier to store them in a CSV file than in a database migrations,
+To populate the database with the maps, run the following command:
+
+```shell
+npm run start
+```
+
+Note: This will clean the `maps` table and insert the maps from the CSV file.
+
+3. Start the backend from the backend folder
+
+```shell
+cargo run
+```
+
+4. Start the frontend from the frontend folder
+
+```shell
+npm run dev
+```
+
+5. Run benchmarks
+
+```shell
+npm run benchmark
+```
+
```

Comment:
Also describe how to manually inspect the tests (maybe in other readme, so that benchmarks/ can eventually be merged). When I open e.g. http://localhost:5173/demo/6 it only shows an empty canvas. When I press "Generate" it only generates 1000 elements (and not 10000 as I would have expected from this test).

Reply:
I skipped an installation step populating the database with maps in the README folder, was it on purpose or by accident? Let me know how could I update the README to indicate its strictness